From 48e8347b4755800287aa653a4d08b89d86bfda52 Mon Sep 17 00:00:00 2001
From: Igor Gnatenko <i.gnatenko.brain@gmail.com>
Date: Mon, 23 Jul 2018 19:25:30 +0200
Subject: [PATCH 1/5] rpm: use %set_build_flags [skip ci]

It has better name and should do the trick.

Signed-off-by: Igor Gnatenko <i.gnatenko.brain@gmail.com>
(cherry picked from commit 99070beec4a6b1ded499405025104483534f97e4)
---
 data/macros.meson | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/data/macros.meson b/data/macros.meson
index 732b68d2..7be47895 100644
--- a/data/macros.meson
+++ b/data/macros.meson
@@ -2,11 +2,7 @@
 %__meson_wrap_mode nodownload
 
 %meson \
-    export CFLAGS="${CFLAGS:-%__global_cflags}"       \
-    export CXXFLAGS="${CXXFLAGS:-%__global_cxxflags}" \
-    export FFLAGS="${FFLAGS:-%__global_fflags}"       \
-    export FCFLAGS="${FCFLAGS:-%__global_fcflags}"    \
-    export LDFLAGS="${LDFLAGS:-%__global_ldflags}"    \
+    %set_build_flags \
     %{__meson}                              \\\
         --buildtype=plain                   \\\
         --prefix=%{_prefix}                 \\\
-- 
2.18.0

